evm/src/NttManager/NttManager.sol

```diff
@@ -241,7 +243,12 @@ contract NttManager is INttManager, RateLimiter, ManagerBase {
     }
 
     /// @inheritdoc INttManager
-    function completeInboundQueuedTransfer(bytes32 digest) external nonReentrant whenNotPaused {
+    function completeInboundQueuedTransfer(bytes32 digest)
+        external
+        nonReentrant
+        whenNotPaused
+        inboundNotPaused
```

Comment:
I think we can safely complete here without pausing on inbound transfers? Or do you think it's better to be aggressive on the pausing here?